requirements_rtd.txt

```diff
@@ -16,6 +15,8 @@ ipython
 jupyter-sphinx>=0.1.2
 pip>=10.0
 annoy
-lightgbm
-xgboost
 sphinx_gallery
+xgboost
+lightgbm
+catboost
+astropy
```

Comment:
Can you undo this, these changes are always confusing when looking back in history